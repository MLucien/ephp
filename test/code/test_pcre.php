<?php
$sujeto = "abcdef";
$patrón = '/def/';
preg_match($patrón, $sujeto, $coincidencias, PREG_OFFSET_CAPTURE);
print_r($coincidencias);

var_dump(preg_match($patrón));
