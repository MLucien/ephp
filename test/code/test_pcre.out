Array
(
    [0] => Array
        (
            [0] => def
            [1] => 3
        )

)

Warning: preg_match() expects at least 2 parameters, 1 given in {{CWD}}/test/code/test_pcre.php on line 7
bool(false)
